src_assets/common/assets/web/index.html

```diff
@@ -89,16 +89,15 @@ <h2>Legal</h2>
       try {
         this.version = (await fetch("/api/config").then((r) => r.json())).version;
         this.githubVersion = (await fetch("https://api.github.com/repos/LizardByte/Sunshine/releases/latest").then((r) => r.json()));
-        if (this.version.split("-g").length > 1) {
+        if (this.version.split(".").length > 1) {
```

Comment:
Needs to be `> 3` or `=== 4`. Previously when splitting with `-g` there would be only one occurrence, so the length was 2... but if checking against `1` with a `.` it will always think it's a nightly version.

Reply:
or `if (!this.buildVersionIsStable()) {`

Reply:
Alternatively, could have a similar method to `buildVersionIsStable` added to check if it's a nightly version.

Reply:
@ReenigneArcher That's basically what I did, I put !this.buildVersionIsStable() and changed the boolean logic to do the version check first, so that it doesn't short circuit and return a false positive.

Basically if you have a boolean check like this

a || b || c

If a is true, then it doesn't test b or c.

So that is why it was important to move the version check as the first condition in the logic basically.